plugins/juju/04known_bugs

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/python3
+import subprocess
+
+from common import helpers, constants, searchtools
+from common.known_bugs_utils import add_known_bug
+from juju_common import (
+    JUJU_LOG_PATH
+)
+
+
+def detect_bug1910958():
+    """Unit fails to start complaining there are members in the relation."""
+    result = {
+        1910958: {
+            "description": ("Unit fails to start complaining there are "
+                            "members in the relation."),
+            "match": ""
+            }
+        }
+    pattern = (
+        r'.* manifold worker returned unexpected error: failed to initialize '
+        'uniter for "[A-Za-z0-9\\-]+": cannot create relation state tracker: '
+        'cannot remove persisted state, relation \\d+ has members'
+    )
+
+    try:
+        s = searchtools.FileSearcher()
+        s.add_search_term(pattern, [0], f"{JUJU_LOG_PATH}/*.log")
+
+        results = s.search()
+
+        disp_text = []
```

Comment:
You can simplify this by tagging the search term with the bug id it corresponds to e.g. https://github.com/dosaboy/hotsos/blob/62d2fc468cd48d6bdc677e520db8d2a41e2f9d4f/plugins/openstack/08agent_checks#L444. That way you can identify hits by tag and only add the known_bug if you get a hit.